tasks/bloom-etl/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Runner, Extractor, Transformer, Loader, defaultMap } from "./src/etl"
+import { DbConfig, Jurisdiction, UrlInfo } from "./src/types"
+import { knex } from "knex"
```

Comment:
@chriscasto  any idea what's happening with this?

Reply:
OK @chriscasto I might know whats happening?

This is the github action. Yarn is only running in the top level dir but I believe you need to `yarn` from inside the tasks/bloom-etl dir to actualy install the packages into node_modules. I would either change yarn at the top level to install this too (I think that @ifranch did this with shared-services? not sure how though), or update the github action to yarn inside your dir.

```
jobs:
  run-linters:
    name: Run linters
    runs-on: ubuntu-latest

    steps:
      - name: Check out Git repository
        uses: actions/checkout@v2

      - name: Set up Node.js
        uses: actions/setup-node@v1
        with:
          node-version: 18

      # ESLint and Prettier must be in `package.json`
      - name: Install Node.js dependencies
        run: yarn install

      - name: Run linters
        run: yarn lint
```

Reply:
I'll try that out.  Thanks!